lightwood/mixers/nn.py

```diff
@@ -634,7 +634,7 @@ def _iter_fit(self, ds, subset_id=None, max_epochs=120000):
                     self.optimizer.step()
                     if self.is_selfaware and self.start_selfaware_training and awareness_loss is not None:
                         awareness_loss.backward(retain_graph=True)
-                        self.selfaware_optimizer.st ep()
+                        self.selfaware_optimizer.step()
```

Comment:
shouldn't a unittest catch this? it was self.selfaware_optmizer.st ep() which seems like it should have broke something (as it looks like a syntax error) ...

Reply:
Yup, it should fail (and it did, locally). The diff GitHub shows here is between the last two commits in the PR, so this error never made it into `stable`.